node/external/transactionAPI/apiTransactionProcessor.go

```diff
@@ -105,12 +115,40 @@ func (atp *apiTransactionProcessor) doGetTransaction(hash []byte, withResults bo
 	return atp.getTransactionFromStorage(hash)
 }
 
-func (atp *apiTransactionProcessor) populateProcessingTypeFields(tx *transaction.ApiTransactionResult) {
+func (atp *apiTransactionProcessor) populateComputedFieldsProcessingType(tx *transaction.ApiTransactionResult) {
 	typeOnSource, typeOnDestination := atp.txTypeHandler.ComputeTransactionType(tx.Tx)
 	tx.ProcessingTypeOnSource = typeOnSource.String()
 	tx.ProcessingTypeOnDestination = typeOnDestination.String()
 }
 
+func (atp *apiTransactionProcessor) populateComputedFieldInitiallyPaidFee(tx *transaction.ApiTransactionResult) {
+	initiallyPaidFee, err := atp.feeComputer.ComputeTransactionFee(tx.Tx, int(tx.Epoch))
+	if err != nil {
+		log.Warn("populateInitiallyPaidFee(): unexpected condition, cannot compute fee", "tx", tx.Hash)
+		return
```

Comment:
so in this case the tx.InitiallyPaidFee is empty as "no fee"?
The return is no different as of the return on L133. How can we know something went wrong other than looking in the logs?

The error is propagated from this call:
feeComputer.createEconomicsInstance(epoch int) which was kind of tested in the constructor already in order to not hit "late construction errors due to programming errors"
Wondering if we can get rid of the error parameter and ease the handling results as seen in this component

Reply:
Fixed to not return an error in `ComputeTransactionFee`, indeed.

Also fixed the outdated log message.